README.md

```diff
@@ -49,6 +49,69 @@ environment-based approach because it keeps sensitive information out of the cod
 
 Once your environment is properly configured, the `Keen` object is ready to go immediately.
 
+### Data Enrichment
+
+A data enrichment is a powerful add-on to enrich the data you're already streaming to Keen IO by pre-processing the data and adding helpful data properties. To activate add-ons, you simply add some new properties within the "keen" namespace in your events. Detailed documentation for the configuration of our add-ons is available `here <https://keen.io/docs/api/ruby#data-enrichment>`.
+
+Here is an example of using the `URL parser <https://keen.io/docs/streams/data-enrichment-overview/#addon-url-parser>`:
```

Comment:
You will want to make sure to use this format for links `[link text](link)`, without the ` marks.